### ISSUE TITLE
Create dark mode feature

### ISSUE THREAD

**Issue by tylla** [erkexzcx/valetudopng#7]
**Is your feature request related to a problem? Please describe.**
More and more apps support dark mode, I'm using OpenHAB and Valetudo in dark mode as well, but the generated image is on a white background which stands out from every user interface.

**Describe the solution you'd like**
I would suggest a feature with a config option to be able to choose between light/dark mode. The simplest implementation should only change the color of the background between white/black.
Maybe the segment colors could be adjusted as well, but maybe I'm overthinking it. :)

**Describe alternatives you've considered**
There's no real alternative, as the image will be used in different places, which are mostly already drawn in dark mode, so a simple image color inversion or other similar techniques wouldn't solve it.

I'm happy to answer any questions that might arise.

**Comment by erkexzcx** [comment]
Hey. Thanks for reporting, but generated map image has no background. I do not set it, and because it's PNG - it's just empty ("transparent").

For example, in HomeAssistant, the background is the same as app's background (dark/light). Or are you talking about segments/rooms colors?

**Comment by tylla** [comment]
I could swear that my images in OpenHAB had white background as well. But now that I double checked, OpenHAB looks perfectly, I see the white background only when viewing the image separately. As it turned out (your reply made me suspicious, so I did some extra research) Firefox has a "feature" to show transparent images on white background regardless of dark mode or not.

The segment colors were thought as an addendum, but now that it turned out that I overlooked the obvious, I think we can let go the segment color adjustment thing, it looks already quite nice. :)

So, my bad... sorry for wasting your time.

And thank you for this nice little piece of software.
